geomstats/sampling/lognormal.py

```diff
@@ -0,0 +1,73 @@
+import geomstats.geometry.spd_matrices as spd
+import geomstats.backend as gs
+
+class LogNormal:
+    """ LogNormal Sampler 
+
+    Parameters:
+    ----------
+    manifold: str, {\'Euclidean\', \'SPDmanifold\'}
+        Name of the Manifold
+    mean: array-like, shape=[..., dim]
+        Mean of the Distribution
+    cov: array-like, shape=[..., dim]
+        Covariance of the Distribution. Should be Positive Semi Definite
+    Returns:
+    --------
+    samples
+
+    Examples:
+    --------
+
+    References:
+    ----------
+    Lognormal Distributions and Geometric Averages of Symmetric Positive Definite Matrices
```

Comment:
Yesss for the reference! See here, for example, to see for how we cite papers in the code:
https://github.com/geomstats/geomstats/blob/909b77b18f2f4c1a8a61f661671412dfa165ee78/geomstats/geometry/discrete_curves.py#L90